lib/bugsnag/api/client/comments.rb

```diff
@@ -2,61 +2,52 @@ module Bugsnag
   module Api
     class Client
 
-      # Methods for the Comments API
+      # Methods for the Errors API
       #
-      # @see https://bugsnag.com/docs/api/comments
+      # @see http://docs.bugsnagapiv2.apiary.io/#reference/comments
       module Comments
-        # List an error's comments
+        # List Comments on an Error
         #
-        # @param error [String] Bugsnag error for which to list comments
-        # @return [Array<Sawyer::Resource>] List of comments
-        # @see https://bugsnag.com/docs/api/comments#list-an-error-s-comments
-        # @example
-        #   Bugsnag::Api.comments("515fb9337c1074f6fd000009")
-        def comments(error, options = {})
-          paginate "errors/#{error}/comments", options
+        # @option offset [Number] The pagination offset (not required for general use)
+        # @option sort [String] Comments are only sortable by creation time e.g. "created_at"
+        # @option direction [String] Which direction to sort the result by, one of: asc, desc
+        # @option per_page [Number] How many results to return per page
+        # @return [Array<Sawyer::Resource>] List of Error Comments
+        # @see http://docs.bugsnagapiv2.apiary.io/#reference/errors/comments/list-comments-on-an-error
+        def comments(project_id, error_id, options = {})
+            paginate "projects/#{project_id}/errors/#{error_id}/comments", options
```

Comment:
Same as collaborators above, the method contents are indented 4 spaces where the class and module declarations are (correctly) 2 spaces.